mantidimaging/core/io/saver.py

```diff
@@ -428,3 +429,34 @@ def make_dirs_if_needed(dirname: Optional[str] = None, overwrite_all: bool = Fal
     elif os.listdir(path) and not overwrite_all:
         raise RuntimeError("The output directory is NOT empty:{0}\nThis can be "
                            "overridden by specifying 'Overwrite on name conflict'.".format(path))
+
+
+def create_rits_format(tof: np.ndarray, transmission: np.ndarray, transmission_error: np.ndarray):
+    """
+    create a RITS format ready for exporting to a .dat file
+
+    :param tof: time of flight
+    :param transmission: transmission value
+    :param transmission_error: transmission_error value
+    :return: RITS format ascii
+
+    """
+    return '\n'.join(
+        ['\t'.join([str(x) for x in row]) for row in zip(tof, transmission, transmission_error, strict=True)])
+
+
+def export_to_dat_rits_format(rits_formatted_data: str, path: Path):
+    """
+    export a RITS formatted data to a .dat file
+
+    :param rits_formatted_data: RITS formatted data
+    :param filepath: path to save the file
+    :param filename: name of the file
+    :return: None
+
+    """
+    if not path.parent.exists():
+        path.parent.mkdir(parents=True)
```

Comment:
I don't think we need this. The file-picker should not let us pick a place not in an existing folder, and if we get a non-existent path it might indicate a problem.